plugins/ecobee/src/main.ts

```diff
@@ -0,0 +1,649 @@
+import axios, { AxiosRequestConfig } from 'axios'
+import sdk, { Device, DeviceInformation, ScryptedDeviceBase, OnOff, DeviceProvider, ScryptedDeviceType, ThermostatMode, Thermometer, HumiditySensor, TemperatureSetting, Settings, Setting, ScryptedInterface, Refresh, TemperatureUnit, HumidityCommand, HumidityMode, HumiditySetting } from '@scrypted/sdk';
+const { deviceManager, log } = sdk;
+
+const API_RETRY = 2;
+
+// Convert Fahrenheit to Celsius, round to 2 decimal places
+function convertFtoC(f: number) {
```

Comment:
I'm guessing this is ecobee API that takes Fahrenheit values. That's very weird!

Reply:
The API actually uses integers to represent decimal numbers (in Fahrenheit). So 720 = 70degF, or 715 = 71.5degF. Weird for sure!